ironic/drivers/modules/redfish/utils.py

```diff
@@ -274,3 +278,100 @@ def _get_system():
                       'node %(node)s. Error: %(error)s',
                       {'address': driver_info['address'],
                        'node': node.uuid, 'error': e})
```

Comment:
These changes are very iDRAC specific, so I think we should create ironic/ironic/drivers/modules/drac/utils.py and move them there.

Reply:
Hi Chris, above method _get_system() in redfish/utils.py is come with redfish code only.  I have moved our methods which are specific to idrac in drac/utils.py file.